package.json

```diff
@@ -118,7 +118,7 @@
     "@types/testing-library__jest-dom": "^5.14.5",
     "@types/uuid": "^9.0.1",
     "@typescript-eslint/eslint-plugin": "^7.5.0",
-    "@typescript-eslint/parser": "^7.5.0",
+    "@typescript-eslint/parser": "8.17.0",
```

Comment:
Why are we updating this package? Should that be done in a different PR?

Reply:
I wasn't able to commit anything until I upgraded this package. If you want, I can add it in a separate PR, but I need it to make future changes to this PR.

Reply:
Interesting, I do see this warning, but it lints successfully. Do you mind adding a caret `^` to the version?

```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
```

Reply:
Upon investigating further, I was getting a Lint error (which wasn't obvious) with something else.

I will add the caret to the version.